From e5575a781813cf2a0af0092761a2c437cd06f28d Mon Sep 17 00:00:00 2001
From: Wes Hayutin <weshayutin@gmail.com>
Date: Fri, 2 Apr 2021 10:36:31 -0600
Subject: [PATCH] exclude redhat-release from yum update in openstack-base

Related to this change I think:
https://git.centos.org/rpms/centos-stream-release/c/ba1b93480d15848e19aa40af070308d6a2a3eb20

Related-Bug: #1922326
Change-Id: I44014883a332c67e9cb9bdd701c717e324e1fc29
(cherry picked from commit 568e40838d8076a997a9a140aa0d1e3bb83a216c)
---
 container-images/tcib/base/base.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/container-images/tcib/base/base.yaml b/container-images/tcib/base/base.yaml
index e5d04d91..0e9131f1 100644
--- a/container-images/tcib/base/base.yaml
+++ b/container-images/tcib/base/base.yaml
@@ -30,7 +30,7 @@ tcib_actions:
 - run: sed -ri '/^(passwd:|group:)/ s/systemd//g' /etc/nsswitch.conf
 - run: dnf install -y {{ tcib_packages['common'] | join(' ') }}
 - run: mkdir -p /openstack
-- run: dnf update -y && dnf clean all && rm -rf /var/cache/dnf
+- run: dnf update --excludepkgs redhat-release -y && dnf clean all && rm -rf /var/cache/dnf
 tcib_cmd: kolla_start
 tcib_entrypoint: dumb-init --single-child --
 tcib_envs:
-- 
2.20.1

